### PR TITLE
VoiceRSS TTS: add support for AAC format

### PR DESCRIPTION
**Pull request by lolodomo**
and fix audio file name extension

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java [hunk 1]
```diff
@@ -106,7 +97,7 @@
     private static Set<String> getSupportedVoices() {
```
Comment (kaikreuzer) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: I am aware that you did not introduce this, but isn't this complete none-sense?
It adds the very same string multiple once x-times (where x is the number of locales) to a Set, which will in the end have a single entry (as all added elements are equal). Would you agree that we can remove this method and simply use a singleton list instead?

File: addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java [hunk 2]
```diff
@@ -30,71 +30,62 @@
...
-    /**
-     * Will support only "MP3" and "OGG" for the moment.
-     */
-    private static Set<String> getSupportedAudioFormats() {
-        Set<String> formats = new HashSet<String>();
-        formats.add("MP3");
-        formats.add("OGG");
-        return formats;
+    private static Set<String> supportedAudioFormats = new HashSet<String>();
+    static {
```
Comment (martinvw) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: Is there a reason you switched to static initializers over methods, imho methods are more readable and common thus better maintainable

File: addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java (hunk 1 above)
Comment (maggu2810) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: `return Collections.singleton("VoiceRSS");` should be fine

Comment (lolodomo) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: I agree.
Will update that

File: addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java (hunk 2 above)
Comment (lolodomo) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: Static initializers looks more readable and simpler to me and are often used in OH / smarthome code.

**Comment by martinvw** [comment]
Forcing a rebuild

**Comment by lolodomo** [comment]
Change done.

File: addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java (hunk 2 above)
Comment (kaikreuzer) [inline addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java]: The advantage here is that the set is not re-constructed on every single method call again, so I am fine with the static block.

**Pull request merged**